spec/requests/user_request_spec.rb

```diff
@@ -177,7 +185,7 @@
       it 'returns meta.errors information', :aggregate_failures do
         error = body.dig('meta', 'errors').first
 
-        expect(error['external_service']).to eq 'VAProfile'
+        expect(error['external_service']).to be_present
```

Comment:
Why does this no longer equal VAProfile?

Reply:
The test originally expected the first error (and only error) to be from the VA Profile veteran status call. After the changes on the PR, the test will generate two errors, one for the demographics and one for the veteran status. I changed it just to verify data is there since the test is only testing for the existence of error metadata in the response.